### ISSUE TITLE
tap action

### ISSUE THREAD

**Issue by home409ca** [custom-cards/bignumber-card#41]
Hi.
Can you add tap action?

**Comment by KHV8** [comment]
Agree, would be a nice feature.

**Comment by smartroad** [comment]
I would love this, I have just posted in the HA forum that I have a "public" facing dashboard and don't want the history pop-up to appear. I tried this card but it does the same. Having a tap being redirectable or disableable would be great!

**Comment by brewing8309** [comment]
+1

**Comment by kerdridge76** [comment]
Yes, a tap action is a great option. I use big number card for wallbox power value. With a tap to url link to evcc were great.

**Comment by tomg1970** [comment]
Tap Action is a very important option.
Thanks

**Comment by Kupie** [comment]
If you're just fine with disabling it, I had the same problem and made a fork years ago:

https://github.com/Kupie/bignumber-card

I'm not skilled enough to make it have customizable actions, though.

**Comment by sxdjt** [comment]
Hello -

I have created a fork of Big Number Card and intend to maintain it as a "continued" version of the original.  I will be releasing the first updated version shortly, but wanted to let you know that I have addressed the issue you have raised.

If you are interested, you can see the current repo here: https://github.com/sxdjt/bignumber-card-continued/

**Comment by KHV8** [comment]
@sxdjt 

Super. I still use Big Number on my main dashboards. Thanks

When you are ready I will move to the new version

EDIT: Can see I can already now download and install. I will move to your version now. Perfect

**Comment by sxdjt** [comment]
Hi @KHV8 -

Sure!  You can grab the current dev version from [my repo](https://github.com/sxdjt/bignumber-card-continued).  Just going to make a couple more clean up passes on it before submitting to HACS.

**Comment by KHV8** [comment]
Works perfectly with tap_action:-) 

Got one problem, the original bignumber-card need to be uninstalled, otherwise my install used the old version. Might be good if you include a small note to remember this.

**Comment by sxdjt** [comment]
Argh - it should have created a "-beta" version of the card so you could test side-by-side.  Sorry about that.  Is the current state OK or do you need the side-by-side?

**Comment by sxdjt** [comment]
For bug discussion in the -continued fork, please use https://github.com/sxdjt/bignumber-card-continued so I can track things there.

**Comment by KHV8** [comment]
I do not need a beta version. Your version works perfectly, after uninstall of the original. 

Also much easier, as you will not have to change anything for already working big-number cards. They just work after install of your version and uninstall of the original

**Comment by sxdjt** [comment]
"Uhh... yeah, it was a feature designed to make transition to the Continued card easier.  Yeah!  That's it..."  :)
